docs/en/get_started.md

```diff
@@ -97,6 +97,13 @@ Note:
    pip install -U git+https://github.com/NVIDIA/MinkowskiEngine -v --no-deps --install-option="--blas_include_dirs=/opt/conda/include" --install-option="--blas=openblas"
    ```
 
+   We also support `Torchsparse` as a sparse convolution backend. If necessary please follow original [installation guide](https://github.com/mit-han-lab/torchsparse#installation) or use `pip` to install it:
+
+   ```shell
+   sudo apt-get install libsparsehash-dev
```

Comment:
we should suggest users to use conda to install libsparsehash, like `conda install -c bioconda sparsehash`, please check whether it works so that we can omit sudo install

Reply:
Add optional command to omit sudo install
```
   或者通过以下安装绕过sudo权限

   conda install -c bioconda sparsehash
   export CPLUS_INCLUDE_PATH=CPLUS_INCLUDE_PATH:${YOUR_CONDA_ENVS_DIR}/include
   pip install --upgrade git+https://github.com/mit-han-lab/torchsparse.git@v1.4.0
```